src/plugins/discourse/mirror.test.js

```diff
@@ -88,27 +90,56 @@ class MockFetcher implements Discourse {
     return matchingLikes.slice(offset, offset + CHUNK_SIZE);
   }
 
+  async topicsBumpedSince(sinceMs: number): Promise<TopicLatest[]> {
+    return Array.from(this._topicToPostIds.keys())
+      .filter((tid) => this._topicCategories.get(tid) != tid)
```

Comment:
I'm confused by this. If topicCategory is a map from topicId to categoryId, why would the topicId and categoryId be the same?

Reply:
I've removed this logic, see #1473